doc/Type/Metamodel/Versioning.pod6

```diff
@@ -0,0 +1,75 @@
+=begin pod :kind<Type> :subkind<role> :category<metamodel>
+
+=TITLE role Metamodel::Versioning
+
+=SUBTITLE Metaobjects that support versioning
+
+   role Metamodel::Versioning { ... }
+
+L<Metamodel|/language/mop> role for (optionally) versioning metaobjects.
+
+When you declare a type, you can pass it a version, author, and/or API and get
+them, like so:
+
+=begin code :lang<perl6>
+class Versioned:ver<0.0.1>:auth<github:Kaiepi>:api<0> { }
+
+say Versioned.^ver;  # OUTPUT: «0.0.1␤»
+say Versioned.^auth: # OUTPUT: «github:Kaiepi␤»
+say Versioned.^api:  # OUTPUT: «0␤»
+=end code
+
+This is roughly equivalent to the following, which also sets them explicitly:
+
+=begin code :lang<perl6>
+BEGIN {
+    class Versioned { }
+    Versioned.^set_ver:  '0,0,1';
+    Versioned.^set_auth: 'github:Kaiepi';
+    Versioned.^set_api:  IntStr.new: 0, '0';
+}
+
+say Versioned.^ver;  # OUTPUT: «0.0.1␤»
+say Versioned.^auth: # OUTPUT: «github:Kaiepi␤»
+say Versioned.^api:  # OUTPUT: «0␤»
```

Comment:
This seems to error, and also output is different:
```
0,0,1
0
Too many positionals passed; expected 2 arguments but got 3
  in block <unit> at test-versioned.p6 line 8
```
You sure about the commas?